### PR TITLE
chore: Added .dockerignore file

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
Closes: #252 

### Describe the changes you've made

Added a `.dockerignore` file, which will reduce the image size and make the image build faster.

## Type of change

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/WebXDAO/WebXDAO.github.io/pull/255"></a>

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+.gitignore
+.gitpod.yml
+.prettierrc 
+.eslintrc.json 
+
+LICENSE
+README.md
+Dockerfile
+CONTRIBUTING.md
+CODE_OF_CONDUCT.md
```
Comment (Panquesito7) [inline .dockerignore:13]: Missing an endline.
```suggestion
CODE_OF_CONDUCT.md

```

Comment (Pradumnasaraf) [reply .dockerignore:13]: We generally don't add a blank line in the `.dockerignore`, no worries added it.

**Review by shuklaritvik06 — APPROVED** [review]
LGTM

**Review by mkubdev — APPROVED** [review]
Hello 👋 Thanks, LGTM

**Review by vinzvinci — APPROVED** [review]
LGTM!

**Pull request merged**